data/ontology/bax_derived_clo.ttl

```diff
@@ -0,0 +1,94 @@
+@prefix bao:   <http://www.bioassayontology.org/bao#> .
```

Comment:
This file is very small: it should be a couple orders of magnitude larger, to incorporate all of the CLO.